src/style.css

```diff
@@ -1,3 +1,104 @@
-body {
+/*Reset default margins*/
+html,
+body,
+div,
+object,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+em,
+img,
+var,
+b,
+u,
+i,
+ol,
+ul,
+li,
+form,
+label,
+legend,
+table,
+tr,
+th,
+td,
+footer,
+header,
+menu,
+nav,
+section {
```

Comment:
I believe that you don't use most of them, just reset those which you actually use

Reply:
Done

Reply:
omg 👀